.dockerignore

```
@@ -0,0 +1,7 @@
+.git*
+.vscode
+.venv
+ansible
+.env
+tests
+todo_app/__pycache__
```

I'd probably go for:
```suggestion
**/__pycache__
```
The fact this is here suggests you are aware that `.dockerignore` has slightly different behaviour to `.gitignore`, in particular in that it _doesn't_ ignore subfolders by default - but your `__pycache__` folders can appear in each folder within the source code folder, so worth covering them all with the `**` prefix to mean arbitrary levels of nesting should be considered for this rule.